config/orchestrations/admission-webhook/0.2/kabanero-operator-admission-webhook-config.yaml

```diff
@@ -117,3 +117,63 @@ webhooks:
     scope: '*'
   sideEffects: Unknown
   timeoutSeconds: 30
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: webhook.operator.kabanero.io
+webhooks:
+- admissionReviewVersions:
```

Comment:
I expected to see this merged in with the `ValidatingWebhookConfiguration` above.  This is a second one, that happens to have the same name as the previous one.